docs/providers/aws/guide/variables.md

```diff
@@ -281,6 +281,9 @@ custom:
 ```
 In this example, we're overwriting the default regex for our variable syntax. So whenever you define variables, you now need to use `${{}}` instead of `${}` (double curly brackets).
 
+## Undefined and null Variables
+Variables that resolve to `undefined` will cause an exception to be thrown when deploying or invoking a function locally. Variables that resolve to `null` will not.  Environment variables with `null` values will not be set for the service.
```

Comment:
Maybe it has to be added with slight modifications for the other providers (I do not have enough experience with them to make a statement there ;-) )